bat/image.go

```diff
@@ -49,7 +49,7 @@ type ImageOptions struct {
 // Image contains all the meta data for a single image
 type Image struct {
 	ImageOptions
-	SizeBytes   int
+	SizeBytes   int `json:"size"`
```

Comment:
Did you consider renaming to Size? This structure is here to encapsulate the output from the CLI.